miniapp/CMakeLists.txt

```
@@ -8,17 +8,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+cmake_minimum_required(VERSION 3.22)
+project(dlaf-miniapps)
```

```suggestion
project(DLAF-miniapps)
```
maybe? Just to keep the all-caps of `DLAF` for the main project.